pkg/controller/core/localqueue_controller.go

```diff
@@ -123,6 +130,9 @@ func (r *LocalQueueReconciler) Create(e event.CreateEvent) bool {
 	}
 	log := r.log.WithValues("localQueue", klog.KObj(q))
 	log.V(2).Info("LocalQueue create event")
+	if ptr.Deref(q.Spec.StopPolicy, kueue.None) != kueue.None {
```

Comment:
`Create` is also called when Kueue restarts. It is possible that this queue had running jobs before Kueue restarted. So this needs to be handled a bit more similar to `Update`.

Reply:
Done